src/fiskaltrust.ifPOS/v1/it/Models/RecMessage.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Runtime.Serialization;
+
+namespace fiskaltrust.ifPOS.v1.it
```

Comment:
Again very epson-specific, but let's keep this for now and see how the other interfaces look, I guess.